backend/message.ts

```diff
@@ -122,14 +140,20 @@ class Processor implements IProcessor {
   updates: UpdateDescr[] = [];
   clearInstanceIds: Set<string> = new Set();
 
-  constructor(public onMessage: OnMessage) {}
+  constructor(public onMessage: OnMessage) { }
```

Comment:
I wonder whether you're using a different version of prettier than I, as I see `{}` changed to `{ }`.